src/main/java/com/laborete/LaboreteAPI/services/UsersService.java

```diff
@@ -0,0 +1,52 @@
+package com.laborete.LaboreteAPI.services;
+
+import com.laborete.LaboreteAPI.entity.Users;
+import com.laborete.LaboreteAPI.exception.ResourceBadRequestException;
+import com.laborete.LaboreteAPI.exception.ResourceServerErrorException;
+import com.laborete.LaboreteAPI.repository.UsersRepository;
+import com.laborete.LaboreteAPI.constants.ConstantsHelper;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class UsersService {
+    private final UsersRepository usersRepository;
+
+    public UsersService(UsersRepository usersRepository) {
+        this.usersRepository = usersRepository;
+    }
+
+    public Users getUserById(UUID uuid) {
+        try {
+            if (uuid == null) {
+                throw new ResourceBadRequestException(ConstantsHelper.uuidIsRequired);
+            }
+            return this.usersRepository.getUsersById(uuid);
+        } catch (Exception e) {
+            throw new ResourceServerErrorException(ConstantsHelper.serverUnavailable);
+        }
+    }
+
+    public Users createUser(Users user) {
+        try {
+            if (user.getFirstName() == null || user.getFirstName().isEmpty()) {
+                throw new ResourceBadRequestException(ConstantsHelper.firstNameIsRequired);
```

Comment:
Suggestion: you can throw standard exception (for instance, `IllegalArgumentException`) and define a global handler for this exception which returns 400 HTTP status. There is practice to use standard exception as much as possible, but it's mention-based.